test/parallel/test-object-proto-accessor-warning.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const common = require('../common');
+
+process.on('warning', common.mustCall());
```

Comment:
There is a `common.expectWarning()` utility.

Reply:
done, but it seems `deprecate(...)` in lib seems to only fire once per dep code so having specialized messages per accessor seems to require unique DEP codes.